packages/core/src/node-utils/framework.ts

```diff
@@ -45,7 +45,7 @@ export function defineFramework({
 }: {
   cwd: string;
   supportedFrameworks?: Frameworks[];
-  manifest?: Manifest;
+  manifest?: Partial<Manifest>;
```

Comment:
why a partial Manifest? You could just send the framework instead, it would be more accurate in that case

Reply:
Was there for when I made the framework mandatory, I'll have to check somethings but there was a part off the init script where either create or patch where called....